index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="grid">
+        <!--Left SideBar-->
+        <div class="left">
+            <!--Company Logo -->
+            <div class="logoObject">
+                <img class="logoImage" src="./image/company.png" alt="Company Logo" />
+                <p class="logoText">Dashboard</p>
+            </div>
+            <div class="flexMenu">
+                <!-- Menu Options -->
+                <div class="flexObject">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                        <title>home-account</title>
```

Comment:
title should be plain english message.
so replace `home-account` with `Home Account`

Reply:
Resolved.